src/smart-components/role/add-role/inventory-groups-role.js

```diff
@@ -101,6 +115,7 @@ const InventoryGroupsRole = (props) => {
 
   const fetchData = (permissions, apiProps) => dispatch(fetchInventoryGroups(permissions, apiProps));
 
+  // eslint-disable-next-line
```

Comment:
I think these might be leftovers.

Reply:
Linter is complaining about the line that follows, easy way to shut it up for now